packages/playground/windows/playground/HostingPane.xaml

```diff
@@ -115,6 +115,16 @@
         VerticalAlignment="Center"
         IsChecked="true"
         Content="Reuse Instance"/>
+      <TextBlock
```

Comment:
Could we make the playground changes in a separate change in master instead of bundling them with the backport?

Reply:
I will submit a separate PR for master, the Playground project changed somewhat so it won't be a straight cherry-pick.

Reply:
That seems reasonable for this change, since it's pretty trivial/low-risk. In general though theirs some nicety of being able to map commits 1:1 between branches . IIRC we've also made changes before in stable branches which we forgot to bring to master.